docs/reference/buildx_history_inspect_attachment.md

```diff
@@ -27,43 +27,155 @@ platform-specific.
 
 ## Examples
 
-### <a name="type"></a> Inspect a provenance attachment from a build (--type)
+### <a name="type"></a> Inspect an attachment by type (--type)
 
-Supported types include `provenance` and `sbom`.
+Supported types include:
+* `index`
+* `manifest`
+* `image`
+* `provenance`
+* `sbom`
+
+#### Index
 
 ```console
-$ docker buildx history inspect attachment qu2gsuo8ejqrwdfii23xkkckt --type provenance
+$ docker buildx history inspect attachment ^0 --type index
```

Comment:
^0 is kind of nonsensical as that would be the default anyway.

Reply:
The command requires at least one arg: https://github.com/docker/buildx/blob/e9b6a01aef29fa2d3affae2b16b77f6a45191b14/commands/history/inspect_attachment.go#L123

```
$ docker buildx history inspect attachment --type index
ERROR: accepts between 1 and 2 arg(s), received 0
```